examples/plotting/wcsaxes_plotting_example.py

```diff
@@ -0,0 +1,65 @@
+"""
+=====================================
+Plotting points on a Map with WCSAxes
+=====================================
+
+This example demonstrates the plotting of point, line and arch in pixel coordinate,
+world coordinate, and SkyCoords respectively when plotting a map with WCSAxes.
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+from numpy.lib.histograms import _ravel_and_check_weights
+
+import sunpy.data.sample
+import sunpy.map
+from sunpy.coordinates.utils import GreatArc
+
+###############################################################################
+# We will start by creating a Map using an AIA 171 image.
+my_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+fig = plt.figure(figsize=(8, 8))
+ax = fig.add_subplot(projection=my_map)
+my_map.plot(axes=ax, clip_interval=(1, 99.9)*u.percent)
+
+###############################################################################
+# Let's first plot a line on the map by using coordinates in arcseconds.
+# The array below `xx` and `yy` are the x and y coordinates that define a from
+# the Sun center (at 0, 0) to the point (500, 500) in arcsecs.
+# The standard plotting in WCSAxes expects pixel coordinates, but we can plot
+# world coordinates (i.e. arcsec) by using the `transform` keyword.
+# Note that the coordinates need to be in degrees rather than arcseconds.
+xx = np.arange(0, 500)
```

Comment:
Hey David, thank you for the review. But `fig, ax = plt.subplots()` doesn't seems to work for me, I am getting an error saying that `TypeError: get_transform() takes 1 positional argument but 2 were given`

Also, could you please check lines 23 and 24

> fig = plt.figure(figsize=(8, 8))
> ax = fig.add_subplot(projection=my_map)

I am sorry if this is vague but to be honest, I didn't get it why we are doing ` fig, ax = plt.subplots()` here, as per my understanding line 23 and 24 is kinda same and doing it's job properly. 
tho i try putting this `fig, ax = plt.subplots()` instead of line 23 and 24, again it's an error. Please guide me through!

Reply:
Hey @nabobalis  Any thoughts on this?

Reply:
Where does the error happen? 

If you have several figures in an example, I believe you will need to create a new figure for it to show up.

Reply:
As per David suggestions i put `fig, ax = plt.subplots()` and ended up getting errors. I believe these below lines are doing the job properly 
```
fig = plt.figure(figsize=(8, 8))
ax = fig.add_subplot(projection=my_map)
```
Do we really need any changes in the codes?

Reply:
Well if the second figure doesn't show up, we need to fix that.

Reply:
Sorry, I forgot that one has to create an axes with the right projection, your lines above look good to me, I will check the example to see if it's plotted correctly.

Reply:
Thank you so much David!